additional-jupyterhub_config.py

```diff
@@ -0,0 +1,97 @@
+def get_ldap_groups(username):
```

Comment:
Why is this in a file if you also put it in a config map in the template?

Reply:
that's where I originally developed it, it doesn't use it